modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -373,6 +373,19 @@ Extra Field Set:: Specifies extra fields.
 Registrations Require Approval:: Select whether registration requires approval.
 Approver ID:: Specifies the approver ID.
 
+== Spec Downloader
```

Comment:
Spec Downloader Versions and Options?

Reply:
On this page each header is the exact name of the component, such as "Spec Downloader", so I'm preserving the current text there.